### DISCUSSION TITLE
i have but copilot oneyear permission bug its expire yesterday

### DISCUSSION THREAD

**Discussion by 2016011969sunyifang** [community/community#80864] [Copilot Conversations]
### Select Topic Area

Bug

### Body

i bought GitHub Copilot at 2023.9
oneyear permission
but now it tell me my Subscription has expired
i dont know why

![Uploading image.png…]()

**Comment by 2016011969sunyifang** [comment]
![1702965586264](https://github.com/community/community/assets/37146715/14bfa9cc-ab43-4f4a-a9ee-93eb380ee0b3)

**Comment by 2016011969sunyifang** [comment]
help me please~

**Comment by 2016011969sunyifang** [comment]
![image](https://github.com/community/community/assets/37146715/ff381b23-0bc1-4466-86f3-6403fab6c2d3)

**Comment by dugiahuy** [comment]
I also have the same issue. Feel disapointed

**Reply by 2016011969sunyifang** [reply]
ye it let me feel sick
